providers/curation/github.js

```diff
@@ -440,27 +470,11 @@ class GitHubCurationService {
       body: `You can review the change introduced to the full definition at [ClearlyDefined](https://clearlydefined.io/curations/${number}).`
     }
     await serviceGithub.issues.createComment(comment)
-    await this._cleanCache(patch.patches)
     return result
   }
 
-  async _cleanCache(patches) {
```

Comment:
FYI, the reason why delete this is because the previous code has tried to delete the cache when the GitHub events come.

Reply:
Makes sense.